### PR TITLE
Homework 10. Home 2. Why Us content and strong-points cards

### PR DESCRIPTION
**Pull request by VioletaScvortova**
Hi! I`m sending my home tasks screenshot for your review. Please, review only "Added Why Us content and strong-points cards" commit. Because the commit "Added Media Queries" loaded automatically, but it belongs to another branch (Home-1).

Lesson 10  Homework Why choose us  Violeta Scvortova
Screenshot_2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,22 +64,55 @@ <h4 class="content__mission">We know how large objects will act, but things on a
...
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog </p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/award.svg" class="card-icon" alt="award image">Web
+                        Development</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
 
+            </div>
```
Comment (VioletaScvortova) [inline index.html:114]: @IvanCeban I added one more </div> here, but unfortunately (I`ve no idea why), it didn't upload.

File: css/style.css [hunk 2]
```diff
@@ -395,19 +381,70 @@ h1,h2,h4 {
...
+    line-height: 20px;
     letter-spacing: 0.2px;
 }
 
+.grid-container {
+    display: grid;
+    align-content: space-around;
+    grid-template-columns: auto auto auto;
+    gap: 30px;
+    background-color: none;
```
Comment (IvanCeban) [inline css/style.css:405]: I don't think you need to specify background: none as there is no default background actually.

File: css/style.css [hunk 3]
```diff
@@ -395,19 +381,70 @@ h1,h2,h4 {
...
+}
+
+
+.card__title {
+    color: #252B42;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    text-align: center;
```
Comment (IvanCeban) [inline css/style.css:428]: as you already set the `text-align` property to `grid-card` you don't need to overwrite it here as well.

By the way, you even don't need to align it center even in `grid-card` as by design it should not be centered.
It's enough to just set it justify on your `card__text` (which you already did)

**Review by IvanCeban — APPROVED** [review]
A couple of css improvements suggested. Nothing critical. 
Overall great job!

File: css/style.css (hunk 2 above)
Comment (VioletaScvortova) [reply css/style.css:405]: +

File: css/style.css (hunk 3 above)
Comment (VioletaScvortova) [reply css/style.css:428]: Took into account. Thank you!

**Pull request merged**